geo-data-server/src/test/java/com/powsybl/geodata/server/dto/LineGeoDataTest.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.geodata.server.dto;
+
+import com.powsybl.iidm.network.Country;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+public class LineGeoDataTest {
+
+    @Test
+    public void test() {
+        LineGeoData lineGeoData = new LineGeoData("l", Country.FR, Country.FR, new ArrayList<>());
+
+        assertEquals("l", lineGeoData.getId());
+        assertEquals(Country.FR, lineGeoData.getCountry1());
+        assertEquals(Country.FR, lineGeoData.getCountry2());
+        assertTrue(lineGeoData.getCoordinates().isEmpty());
+
+        LineGeoData.LineGeoDataBuilder lineGeoDataBuilder  =  LineGeoData.builder();
+        lineGeoDataBuilder.id("testId");
+
+        assertEquals("LineGeoData.LineGeoDataBuilder(id=testId, country1=null, country2=null, coordinates=null)", lineGeoDataBuilder.toString());
```

Comment:
Why do you test the builder instance and not the built object?

Reply:
i should test both of them.
if i don't test toString method, coverage for that class decrease from 100% to 56% !